cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleSpecification.java

```diff
@@ -59,36 +60,26 @@ public Schedule getSchedule() {
   }
 
   @Override
-  public boolean equals(Object o) {
-    if (this == o) {
```

Comment:
Changed to use internal (in `cdap-api`) `Objects`. This change was needed b/c in the list adapters endpoint, some of the `ScheduleSpecification` fields can be empty.